### PR TITLE
presto 0.5.4 has incompatibilities. 0.6.2 should be used

### PR DESCRIPTION
**Pull request by aerijman**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR updates an existing tool or tool collection

### CODE REVIEW

File: tools/presto/prestor_abseq3.xml [hunk 1]
```diff
@@ -72,7 +72,7 @@ report_abseq3('.', sample='${sample}', output_dir='.', output_file='report.html'
     <help><![CDATA[
 Creates an HTML QC report from the log files generated when running pRESTO.
 
-See the `Prestor project page <https://bitbucket.org/javh/prototype-prestor/src>`_ for more information.
```
Comment (bgruening) [inline tools/presto/prestor_abseq3.xml:75]: @aerijman is that correct? Is presto the same as prestor?

Comment (aerijman) [reply tools/presto/prestor_abseq3.xml:75]: @bgruening 
1. Thank you for dealing with this.
2. No. `prestor_abseq3` just generate simple outputs to visualize the results of `pResto`. I don't know what presto`r` would be (my guess is presto-Report) and I also don't know if there is an alternative link that works for that. Can't we just take that line out? Or include a link to presto (https://bitbucket.org/kleinstein/presto/src/master/)?

presto 0.5.4 uses a feature of Biopython that does not exists anymore and it crashes. 


Thanks a lot!!

**Comment by aerijman** [comment]
@bgruening, is it ready to merge?

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Yes :) 

Thanks a lot!

**Pull request merged**